include/osg/Program

```diff
@@ -252,7 +252,19 @@ class OSG_EXPORT Program : public osg::StateAttribute
         typedef std::map<std::string, UniformBlockInfo> UniformBlockMap;
 
         //const UniformBlockMap& getUniformBlocks(unsigned contextID) const;
-    public:
```

Comment:
This file is missing a change to use flat_map. Might have been added after the last time you merged in makOsg.